plugins/lsp/lsp-status.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "lsp-status";
+  originalName = "lsp-status.nvim";
+  defaultPackage = pkgs.vimPlugins.lsp-status-nvim;
+  maintainers = [ helpers.maintainers.b3nb5n ];
+  settingsOptions =
+    let
+      indicator_opt =
+        default:
+        helpers.defaultNullOpts.mkStr default ''
+          The string to show as diagnostics.
+          If you don't have Nerd/Awesome Fonts you can replace defaults with ASCII chars.
+        '';
+    in
+    {
+      kind_labels = helpers.defaultNullOpts.mkAttrsOf types.str "{}" ''
+        An optional map from LSP symbol kinds to label symbols. Used to decorate the current function name.
+      '';
+
+      select_symbol = helpers.defaultNullOpts.mkStr "" ''
+            An optional handler of the form function(cursor_pos, document_symbol) that should return true if document_symbol (a DocumentSymbol) should be accepted as the symbol currently containing the cursor.
+        	'';
```

Comment:
Actually, we also need to wrap any code snippets in backticks (especially snippets containing underscores, since that's markdown syntax for "italic"):

```suggestion
      select_symbol = helpers.defaultNullOpts.mkStr "" ''
            An optional handler of the form `function(cursor_pos, document_symbol)` that should return
            `true` if `document_symbol` (a `DocumentSymbol`) should be accepted as the symbol currently
            containing the cursor.
       '';
```